datastoretest/mongo.go

```diff
@@ -77,6 +82,14 @@ func (db *DB) Clean() {
 	}
 }
 
+// Drop removes the entire database including all of its collections.
+func (db *DB) Drop() {
```

Comment:
Client of godb can forget to invoke Drop method. Therefore i will invoke drop database in Client method.